bili.config.js

```diff
@@ -1,11 +1,13 @@
 module.exports = {
   banner: true,
-  moduleName: "mdDaterangePicker",
-  name: "md-daterange-picker",
+  // TODO module name 与 name 的关系？(difference between module name and name ??)
+  // https://github.com/ly525/md-date-range-picker/issues/2
+  moduleName: "mdDateRangePicker",
+  name: "md-date-range-picker",
   format: ["umd", "umd-min", "cjs", "es"],
```

Comment:
```suggestion
  format: ["umd", "umd-min", "cjs", "es"],
```
need to be clear about these package format and the usage scenarios
 这部分需要研究一下，打包出来的各种格式有何异同以及使用场景。